crates/holochain/CHANGELOG.md

```diff
@@ -7,7 +7,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## Unreleased
 - **BREAKING**: As the DPKI feature is unstable and incomplete, it is disabled with default cargo features and put behind a feature called `unstable-dpki`. If this feature is specified at compile time, DPKI is enabled by default.
-- **BREAKING**: Conductor::get_dna_definitions now returns an `IndexMap` to ensure consistent ordering.
+- **BREAKING**: Issuing and persisting warrants is behind a feature `unstable-warrants` now. Warrants have not been tested extensively and there is no way to recover from a warrant. Hence the feature is considered unstable and must be explicitly enabled. 
```

Comment:
I'd also mention that warrants get stored in the Actions table and some queries get broken when warrants are stored

Reply:
So that doesn't work right now?

Reply:
I mean when a warrant action is stored, does a part of Holochain break is how I understand this?

Reply:
Yes, the `query` function which retrieves your source chain will just give you a deserialisation error

Reply:
I'm assuming that's probably not all,  it's just all I know about from Holo testing so far

Reply:
Oh, that's pretty bad and another reason to put this behind an unstable feature.